community/font-awesome/build

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh -e
+
+install -Dm644 ./otfs/*.otf -t "$1/usr/share/fonts/OTF"
```

Comment:
The `-t` flag for `install` isn't very portable. This also works:

```
mkdir -p "$1/usr/share/fonts/OTF"
cp ./otfs/*.otf "$1/usr/share/fonts/OTF"
```

Reply:
Without `install -Dm644` command?

Reply:
Yes

Reply:
Done